src/detect.c

```diff
@@ -292,6 +292,69 @@ char *DetectLoadCompleteSigPath(const DetectEngineCtx *de_ctx, char *sig_file)
     return path;
 }
 
+static SigString *SigStringAlloc(void)
+{
+    SigString *sigstr = SCMalloc(sizeof(SigString));
```

Comment:
I prefer SCCalloc as then the structure is initialized to 0. Now only 2 of the 4 fields are initialized.